### DISCUSSION TITLE
Is Pleccer still being developed?

### DISCUSSION THREAD

**Discussion by Trent54** [pleccer/Pleccer#6] [Q&A]
Hi, I've been using Pleccer-2.6.0-beta1_2023-03-08 based on Slic3r for several months now on MacOS. I really enjoy it, though I'm not sure I fully understand all of the cool settings, and it does odd stuff sometimes (bug or feature? idk!). Is this software still being developed?

Thanks!

**Comment by Jan-Snijder** [comment]
Long time waiting for an answer, okay the a little kick to the top then :)
